### PR TITLE
Expansion operator

### PR DESCRIPTION
**Pull request by dakujem**
1. Added docs for `(expand)` operator:
   - en
   - cs

2. Cosmetic change in readme wording

### CODE REVIEW

**Comment by dakujem** [comment]
This update is a reaction to this **Latte** PR: https://github.com/nette/latte/pull/157

**Comment by dg** [comment]
I merged your PR #158 so it is not anymore true that n:attr requires `(expand)`.

**Comment by dakujem** [comment]
It will take some time before the change is released though. Propose a solution please.

My intention was for the `(expand)` operator to be properly mentioned and documented. Even after the mentioned change it is still useful for registering variables in included templates/blocks using the `{include}` macro.

**Comment by dg** [comment]
> It will take some time before the change is released though. Propose a solution please.

Sure, but because the problem with the array & `n:attr` is very rare (no one has previously reported it), there is no need to demonstrate it just on it. An example with `{include}` is quite sufficient.

File: latte/cs/macros.texy [hunk 1]
```diff
@@ -776,4 +796,18 @@ Dumpování proměnných `{dump}` .{toc: Dumpování proměnných}
 {dump}       {* Vypíše všechny aktuálně definované proměnné *}
 \--
 
+
+
+Operátor rozbalení pole `(expand)` .{toc: Operátor rozbalení}
+----------------
+
+Operátor `(expand)` zabraňuje obalení argumentu prázdným polem v makrach, které očekávají rozbalené pole, jako  `{include}` a `n:attr`.
+Je užitečný pro posílání argumentů do vkládaných bloků nebo šablon a pro generování více atributů z pole.
```
Comment (dg) [inline latte/cs/macros.texy]: Operátor `(expand)` umožňuje použití pole v místě, kde se očekává více argumentů. Jde o obdobu operátoru `...` z PHP (odkaz na http://php.net/manual/en/functions.arguments.php#functions.variable-arg-list.new), na rozdíl od něj zachovává i klíče.

Lze jej použít například pro předání argumentů do bloků nebo vkládaných šablon, pokud je máme v poli.

**Comment by f3l1x** [comment]
@dakujem Could you please rebase it and resolve all @dg 's points? :-)

**Comment by f3l1x** [comment]
ping @dakujem

**Pull request merged**